### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by robmoffat**

### CODE REVIEW

**Comment by robmoffat** [comment]
@Alexstpierrework

File: Readme.md [hunk 1]
```diff
@@ -2,79 +2,113 @@
...
+
+### FINOS Code of Conduct
+
+Participants of FINOS standards projects should follow the FINOS Code of Conduct, which can be found at: <https://community.finos.org/docs/governance/code-of-conduct>
 
-FINOS Common Cloud Controls is maintained and run through GitHub. Simply [Raise a GitHub Issue](https://github.com/finos/common-cloud-controls/issues/new/choose) to ask questions or make suggestions. 
+## Governance
 
-## FINOS CCC Steering Committeee Members
+### FINOS CCC Steering Committeee Members
```
Comment (sshiells-scottlogic) [inline Readme.md:97]: Can we remove the extra e from the end of committee?

**Review by sshiells-scottlogic — CHANGES_REQUESTED** [review]
Appreciate this ask isn't part of your changes but feels like a good time to fix

**Review by sshiells-scottlogic — APPROVED** [review]
Thanks for the update. Looks good to me.

File: docs/governance/finos-poc.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# FINOS Point-of-Contact (POC)
```
Comment (sshiells-scottlogic) [inline docs/governance/finos-poc.md:1]: In some places we link to the FINOS POC section here: https://github.com/finos/common-cloud-controls/blob/main/docs/governance/steering/charter.md#finos-point-of-contact

Should these links be updated to point to this page?

Comment (sshiells-scottlogic) [reply docs/governance/finos-poc.md:1]: Added

**Comment by eddie-knight** [comment]
Very confused to see duplication reduction listed as an owner 🤔 Doesn't match what I see in the CODEOWNERS file.

File: Readme.md [hunk 3]
```diff
@@ -2,79 +2,113 @@
...
 
 The CCC Steering Committee is the governing body of the CCC project, providing decision-making and oversight pertaining to the CCC project bylaws, sub-organizations, and financial planning. The Steering Committee also defines the project values and structure. [Documented here](docs/governance/steering/charter.md).
 
-| FINOS CCC Maintainer | Representing   | Seat |
-| -------------------- | -------------- | ---- | 
-| Jon Meadows          | Citi           | FSI  |
-| Oli Bage             | LSEG           | FSI  |
-| Simon Zhang          | BMO            | FSI  |
-| Paul Stevenson       | Morgan Stanley | FSI  |
+| FINOS CCC Maintainer | Representing   | Seat      |
```
Comment (eddie-knight) [inline Readme.md:101]:
```suggestion
| Name                 | Representing   | Seat      |
```

Feels like "maintainer" is opening more questions than it answers here

**Review by jared-lambert — APPROVED** [review]
Nice updates.

File: Readme.md [hunk 4]
```diff
@@ -2,79 +2,113 @@
...
 This standard is a collaborative project which aims to develop a unified set of cybersecurity, resiliency, and compliance controls for common services across the major cloud service providers (CSPs).
 
-You can read more and register your interest on [finos.org/common-cloud-controls-project](https://www.finos.org/common-cloud-controls-project).
+## What Are The Benefits?
 
-## FINOS CSLA Needed to Participate in Common Cloud Controls
+#### 💯 Defining Best Practices Around Cloud Security
 
-All FINOS Common Cloud Controls participants are required to sign a FINOS [Community Specification Contributor License Agreement](https://github.com/finos/standards-project-blueprint/blob/main/governance-documents/Getting%20Started.md#best-practices) before joining project calls and collaborating in working groups.
+> CCC aims to standardize cloud security controls for the banking sector, providing a common set of controls that CSPs can implement to meet the requirements of FS firms. As multiple FS firms are involved in the project, effort is shared, the controls will be representative of the sector as a whole, and be more robust than any one firm could develop on its own.
```
[the remark this answers is not in the document]
Comment (robmoffat) [reply Readme.md:15]: this style came from https://github.com/finos/morphir?tab=readme-ov-file#what-are-the-benefits originally, but I agree, it's an abuse of markdown and we can get rid of it

File: Readme.md [hunk 5]
```diff
@@ -2,79 +2,113 @@
...
+
+### 3. Raise a FINOS Common Cloud Controls GitHub Issue
+
+FINOS Common Cloud Controls is maintained and run through GitHub. Simply [Raise a GitHub Issue](https://github.com/finos/common-cloud-controls/issues/new/choose) to ask questions or make suggestions.
+
+### FINOS CSLA Needed to Participate in Common Cloud Controls
+
+All FINOS Common Cloud Controls participants are required to sign a FINOS [Community Specification Contributor License Agreement](https://github.com/finos/standards-project-blueprint/blob/main/governance-documents/Getting%20Started.md#best-practices) before joining project calls and collaborating in working groups.
+
+Please visit [participants.md](participants.md) and raise a Pull Request by adding your `name`, `organisation` and `enrollment date` to the markdown file.
```
[the remark this answers is not in the document]
Comment (robmoffat) [reply Readme.md:78]: Yeah, I've seen this before!   

Members should really refer to FINOS members (i.e. Gold, Silver, Platinum etc) whereas participants are people who are engaged in the project/meetings/github etc.

File: Readme.md [hunk 6]
```diff
@@ -2,79 +2,106 @@
...
+Email help@finos.org if you require further help.
+
+### FINOS Code of Conduct
+
+Participants of FINOS standards projects should follow the FINOS Code of Conduct, which can be found at: <https://community.finos.org/docs/governance/code-of-conduct>
+
+## Governance
 
-## FINOS CCC Steering Committeee Members
+### FINOS CCC Steering Committee Members
```
Comment (robmoffat) [inline Readme.md:90]:
```suggestion
### FINOS CCC Steering Committee
```

File: Readme.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (robmoffat) [reply Readme.md:78]: Have removed the usage of the word "member" from the README at least.

**Comment by robmoffat** [comment]
Well done @jared-lambert on getting through the CCLA process!

File: Readme.md [hunk 7]
```diff
@@ -2,79 +2,106 @@
 
 <img height="100px" src="https://github.com/finos/branding/blob/master/project-logos/active-project-logos/FINOS%20Common%20Cloud%20Controls%20Logo/Horizontal/2023_FinosCCC_Horizontal.svg?raw=true"/>
 
-FINOS Common Cloud Controls (FINOS CCC) is the codename for an open standard project, originally proposed by Citi and currently incubating in FINOS, to describe consistent controls for compliant public cloud deployments in the financial services sector.
+## What Is It?
+
+FINOS Common Cloud Controls (FINOS CCC) is the codename for an open standard project, originally proposed by Citi and currently incubating in FINOS, to describe consistent controls for compliant public cloud deployments in the financial services (FS) sector.
```
Comment (eddie-knight) [inline Readme.md:7]:
```suggestion
FINOS Common Cloud Controls (FINOS CCC) is an open standard project that describes consistent controls for compliant public cloud deployments in the financial services (FS) sector.
```

**Comment by robmoffat** [comment]
> Waiting on code owner review from finos/ccc-steerco

who is this?

**Review by eddie-knight — APPROVED** [review]
LGTM! Thanks @robmoffat!

**Pull request merged**